_health-care/_js/_components/date_input.jsx

```diff
@@ -28,31 +28,31 @@ class DateInput extends React.Component {
 
   render() {
     return (
-        <div className="usa-date-of-birth">
-          <div className="usa-datefield usa-form-group usa-form-group-month">
-            <label htmlFor={this.id + "-month"}>Month</label>
-            <input id={this.id + "-month"} aria-describedby="dobHint"
-                className="usa-form-control" pattern="0?[1-9]|1[012]"
-                type="number" min="1" max="12" value={this.state.month}
-                onChange={this.handleMonthChange} />
-          </div>
-          <div className="usa-datefield usa-form-group usa-form-group-day">
-            <label htmlFor={this.id + "-date"}>Day</label>
-            <input id={this.id + "-date"} aria-describedby="dobHint"
-                className="usa-form-control" pattern="0?[1-9]|1[0-9]|2[0-9]|3[01]"
-                type="number" min="1" max="31" value={this.state.date}
-                onChange={this.handleDateChange}  />
-          </div>
-          <div className="usa-datefield usa-form-group usa-form-group-year">
-            <label htmlFor={this.id + "-year"}>Year</label>
-            <input id={this.id + "-year"} aria-describedby="dobHint"
-                className="usa-form-control" pattern="[0-9]{4}" type="number"
-                min="1900" max={new Date().getFullYear()}
-                value={this.state.year} onChange={this.handleYearChange} />
-          </div>
+      <div className="usa-date-of-birth">
+        <div className="usa-datefield usa-form-group usa-form-group-month">
+          <label htmlFor={this.id + "-month"}>Month</label>
+          <input id={this.id + "-month"} aria-describedby="dobHint"
+              className="usa-form-control" pattern="0?[1-9]|1[012]"
+              type="number" min="1" max="12" value={this.state.month}
+              onChange={this.handleMonthChange} />
```

Comment:
The only change here was that this was indented one too many things, so I just unindented it once.